news/environment_fixes.rst

```diff
@@ -0,0 +1,30 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* xonsh/environ.py: new rule: if default for "registered" environment variable is set to ``DefaultNotGiven``_ 
```

Comment:
I think that the trailing underscore here should be removed in the in-line code

Reply:
ok